src/transformers/models/ctrl/modeling_tf_ctrl.py

```diff
@@ -816,9 +816,8 @@ def call(
                 batch_size, sequence_length = shape_list(input_ids)[:2]
             else:
                 batch_size, sequence_length = shape_list(inputs_embeds)[:2]
-            assert (
-                self.config.pad_token_id is not None or batch_size == 1
-            ), "Cannot handle batch sizes > 1 if no padding token is defined."
+            if self.config.pad_token_id is None or batch_size != 1:
```

Comment:
```suggestion
            if self.config.pad_token_id is not None and batch_size != 1:
```
Careful, the negation is wrong here :-)